src/Doctrine/DBAL/Types/UuidArrayType.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Contao\CoreBundle\Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Types\BlobType;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+/**
+ * Type that maps an array of UUIDs to a BLOB SQL type.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ * @link https://blog.vandenbrand.org/2015/06/25/creating-a-custom-doctrine-dbal-type-the-right-way/
+ */
+class UuidArrayType extends BlobType
+{
+    const UUID_ARRAY = 'uuid_array';
+
+    /**
+     * Converts the serialized array of binary UUIDs to string representation.
+     *
+     * @param mixed            $value
+     * @param AbstractPlatform $platform
+     *
+     * @return string
+     */
+    public function convertToPhpValue($value, AbstractPlatform $platform)
+    {
+        if (null === $value || !is_string($value)) {
+            return null;
+        }
+
+        $value = @unserialize($value);
```

Comment:
It might be available if the framework is booted. Or we could boot the framework. But we know that's somewhat wrong :D

A secure Doctrine array (one that does not allow objects) is not specific for Contao. I know Joomla had similar security issues with unserializing object (from the session though).